.github/workflows/lecture_participation.yml

```diff
@@ -0,0 +1,36 @@
+name: Lecture Participation Information
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  issues: write
+
+jobs:
+  track-participation:
+    if: github.event.issue.title == 'Lecture participation tracker 2024'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Setup Python
+        uses: actions/setup-python@v2.2.1
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          if [ -f ./tools/requirements.txt ]; then pip install -r ./tools/requirements.txt; fi
+
+      - name: Set timezone to Europe/Stockholm
+        run: sudo timedatectl set-timezone Europe/Stockholm
```

Comment:
Could definitely be done in the Python script instead of using system time, in turn set in the workflow. It's another remnant from assignment-statistics.yml, but probably cleaner to just do it in the script.